app/concepts/stellar_base/deposit_requests/operations/create.rb

```diff
@@ -54,6 +54,9 @@ def set_defaults!(options, params:, **)
           options["model"].min_amount = 0.0
 
           # Make Deposits free unless we want it configured
+          #
+          # TODO: this should come from dest_extra
```

Comment:
I think `dest_extra` is just for withdrawals right?

Reply:
ay, yeah, let me fix that